Test-NetStack.psm1

```diff
@@ -125,8 +125,8 @@ Function Test-NetStack {
         [Parameter(Mandatory = $false, ParameterSetName = 'OnlyPrereqIPTarget'   , position = 1)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesNodes'   , position = 1)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesIPTarget', position = 1)]
-        [ValidateSet('1', '2', '3', '4', '5', '6')]
-        [Int32[]] $Stage = @('1', '2', '3', '4', '5', '6'),
+        [ValidateSet('1', '2', '3', '4', '5', '6', '7')]
+        [Int32[]] $Stage = @('1', '2', '3', '4', '5', '6', '7'),
```

Comment:
Are we requiring this for a general pass? Or is this for a specific testing purpose? If specific testing purpose, $Stage should not specify stage 7 (which indicates the automatic/default list of stages to run)